modules/virt-creating-custom-resources-hpp.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-custom-resources-hpp_{context}"]
+= Creating host path provisioner custom resources
+
+If you upgrade to {VirtProductName} version 4.10 on an existing cluster, the system will:
+* Update the legacy host path provisioner (HPP)
+* Create the CSI driver provisioner based on the contents of the legacy HPP custom resource.
+
+In order to use the CSI driver, you must create an appropriate storage class.
+
+If you are installing {VirtProductName} version 4.10 on a new cluster, you must install a CSI driver custom resource.
+
+== Creating the CSI driver provisioner custom resource
+
+The legacy provisioner and the CSI driver provisioner are supported in parallel for a number of releases. However, at some point, the legacy provisioner will no longer be supported. To prepare for this change,
+create a CSI driver custom resource using a basic storage pool.
+
+Storage pools allow you to specify the source and path that are used by the CSI driver. They also allow you to name the storage pool that is used in the storage class.
+
+.Procedure
+
+. Create a YAML file for the CSI driver provisioner custom resource. For example:
++
+[source,terminal]
+----
+$ touch hostpathprovisioner_cr.yaml
+----
+
+. Edit the file. For example:
++
+[source,yaml]
+----
+apiVersion: hostpathprovisioner.kubevirt.io/v1beta1
+kind: HostPathProvisioner
+metadata:
+ name: hostpath-provisioner
+spec:
+ imagePullPolicy: IfNotPresent
+ storagePools: <1>
+   - name: <any_pool>
+     path: "</var/myvolumes>" <3>
+ workload:
+   nodeSelector:
+     kubernetes.io/os: linux
+----
+<1> The `storagePools` specification is an array to which you can add multiple entries.
+<2> Read/write access required.
```

Comment:
If the above should be `<2>` then this should say that this is the path used on the node to create directories under.

Reply:
Changed <2> to: 

 Create directories under this node path. Read/write access required. Ensure that the node-level directory (`/var/myvolumes`) is not on the same partition as the operating system. If it is, users of the volumes can potentially fill the operating system partition and cause the node to impact performance, become unstable, or become unusable.